app/controllers/authenticate_controller.rb

```diff
@@ -200,7 +200,10 @@ def handle_authentication_error(err)
 
   def log_backtrace(err)
     err.backtrace.each do |line|
-      logger.debug(line)
+      # We want to print a minimal stack trace in INFO level so that it is easier
+      # to understand the issue. We still want to print the full stack trace
+      # (including the ryb gems code) so we print it in DEBUG level
```

Comment:
```suggestion
      # (including the ruby gems code) so we print it in DEBUG level
```

It might also be helpful to elaborate in the comment that this filters the trace output to only Conjur application code, and not code from the Gem dependencies.

Reply:
Done